doc/user/content/ingest-data/mysql/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "MySQL"
+description: "Connect and ingest data from MySQL."
+---
```

Comment:
Later on, may use a section template in the docs, but for this work, just using the regular single page template.